train.py

```diff
@@ -25,15 +24,13 @@ def train(net, trainloader, config, criterion, optimizer, use_GPU):
             X, S1, S2, labels = data
             if X.size()[0] != config.batch_size:
                 continue  # Drop those data, if not enough for a batch
-            # Send Tensors to GPU if available
-            if use_GPU:
-                X = X.cuda()
-                S1 = S1.cuda()
-                S2 = S2.cuda()
-                labels = labels.cuda()
-            # Wrap to autograd.Variable
-            X, S1 = Variable(X), Variable(S1)
-            S2, labels = Variable(S2), Variable(labels)
+            # automaticlly select device, device agnostic 
```

Comment:
Please capitalize the first letter of the comment.

Reply:
Hi @kentsommer , thanks for you reply, I solved at commit: [b18247](https://github.com/Kelvinson/pytorch-value-iteration-networks/commit/b1824f7a361efde199a0fc39a323767f08cf3b34) ,can you merge it now? Thank you and wish you a happy new year!